olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/SeekRequest.h

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (C) 2020 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+#pragma once
+
+#include <vector>
+
+#include <olp/dataservice/read/model/StreamOffsets.h>
+
+namespace olp {
+namespace dataservice {
+namespace read {
+
+/**
+ * @brief Request class used to seek offset for a stream layer.
+ */
+class DATASERVICE_READ_API SeekRequest final {
+ public:
+  using StreamOffsets = model::StreamOffsets;
+
+  /**
+   * @brief Sets offsets for the request.
+   *
+   * An empty by default.
```

Comment:
```suggestion
   * No offset by default.
```
`No offset` is that what you mean by `An empty`? Then I suggest the change as above.